assets/scss/layouts/_header.scss

```diff
@@ -27,7 +27,7 @@
 */
 
 .navbar {
-  z-index: 1000;
+  z-index: 1021;
```

Comment:
![image](https://user-images.githubusercontent.com/29014573/201862104-e4e3d491-5467-4e7c-abc8-d13d085027b1.png)

The dropdown with the languages is not visible because the subnav holding the searchbar has greater z-index. That is why I increased the z-index of the navbar. I tried to add greater z-index of the dropdown but it seems that it was invisible again and I don't know why.